Cargo.toml

```diff
@@ -8,13 +8,19 @@ license = "MIT OR Apache-2.0"
 members = ["", "fuzz"]
 
 [workspace.dependencies]
```

Comment:
Do we really need a workspace here?

Reply:
It's because [some deps are shared with the private `fuzz` crate](https://github.com/near/near-account-id/blob/main/fuzz/Cargo.toml). Easier to keep it all in sync. If you think it's just too verbose and not worth it, I can go for no workspace.

Reply:
Well, it was not obvious and I feel it creates more confusion than it solves 2 lines of version syncing for fuzz.